### DISCUSSION TITLE
Marvel Cinematic Universe issues

### DISCUSSION THREAD

**Discussion by jarkle** [meisnate12/Plex-Meta-Manager-Configs#135] [General]
There seems to be some issues with this collection. Pretty sure I have most of the movies but only 8 appear and they don't seem accurate..

![Screenshot_20220214-184127_Plex](https://user-images.githubusercontent.com/10996990/153970884-44182027-11be-4884-bc5a-6cb518bda588.jpg)

**Comment by meisnate12** [comment]
i mean you gotta include a log and maybe the collection that's running or the file your running this doesn't tell me much
